dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeAPIImpl.java

```diff
@@ -362,6 +379,23 @@ public List<MultiTree> copyPersonalizationForPage (final String pageId,
         return multiTrees;
     } // copyPersonalizationForPage.
 
+    @Override
+    public List<MultiTree> copyVariantForPage (String pageId, String baseVariant, String newVariant) throws DotDataException{
```

Comment:
set to final

Reply:
done https://github.com/dotCMS/core/pull/23258/commits/be0dccb64465aa648a685a47be69411aa478f1b7#diff-ee5026536e73ba67dcf187415db7af51b4ea21727f9a73b52ce145bbea53b174R110